haskell-src/exec/Chainweb/Listen.hs

```diff
@@ -67,8 +70,18 @@ getOutputsAndInsert env ph@(PowHeader h _) = do
              (hashB64U $ _blockHeader_hash h)
       logg Info $ fromString $ show e
     Right pl -> do
+      let chain = _blockHeader_chainId h
+          height = _blockHeader_height h
+          hash = _blockHeader_hash h
+          tos = _blockPayloadWithOutputs_transactionsWithOutputs pl
+          ts = S.fromList $ map (\(t,tout) -> mkTxSummary chain height hash t tout) tos
+          msg = printf "Got new header on chain %d height %d" (unChainId chain) height
+          addendum = if S.null ts then "" else printf " with %d transactions" (S.length ts)
+      when logTxSummaries $ do
+        logg Info $ fromString $ msg <> addendum
+        mapM_ (logg Info . fromString . show) ts
```

Comment:
I see two behavior differences here:
1) We're logging at the `Info` level instead of the `Debug` level performed by the old `serverHeaderHandler`.
2) We used to log the full `transactionsWithOutputs` (i.e. `tos`), but this new code switched to logging `ts`, the `TxSummary`.

I think the following would bring the changes in line:
```suggestion
        logg Debug $ fromString $ msg <> addendum
        forM_ tos $ \txWithOutput -> 
          logg Debug $ fromString $ show txWithOutput
```